plugins/modules/cloud/linode/linode.py

```diff
@@ -517,10 +526,7 @@ def linodeServers(module, api, state, name,
                 instance['password'] = password
             instances.append(instance)
 
-    elif state in ('stopped'):
```

Comment:
Wow, that module was in a pretty horrible state...

Reply:
I don't claim to have put it in a good one yet, but it's a start.